File: Exercice2/battlesimulator/src/main/scala/com/graygzou/Engine/Launcher3D.scala
/**
  * @author Grégoire Boiron <gregoire.boiron@gmail.com>
  * @version 0.0.1
  */

package com.graygzou.Engine

object Launcher3D {

  /**
    * Main function. Call the launch a fight
    * @param args
    */
  def main(args: Array[String]): Unit = {

    val game = new BattleSimulation3D()

    game.initScalaContext("","")

    game.cleanScalaContext

  }

}

File: Exercice2/battlesimulator/src/main/java/Engine/Launcher3D.java
/**
 * @author : Grégoire Boiron <gregoire.boiron@gmail.com>
 * @version 0.0.1
 */

package Engine;

/** This class call the JME3 engine to render the fight in 3D
 * It has to be in Java since the engine need specific method
 *
 * @author Grégoire Boiron <gregoire.boiron@gmail.com>
 * @version 0.0.1
 */
public class Launcher3D {

    public static void main(String[] args){
        Engine3D game = new Engine3D();
        game.start(); // start the game
    }

}